Views/JSuPCB/Delete.cshtml

```diff
@@ -0,0 +1,80 @@
+@model SUMBER.Models.Sumber.JSuPCB
+
+@{
+    ViewData["Title"] = "Jadual PCB - Hapus";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+<partial name="_JSRadio" />
```

Comment:
Change name '_JSRadio' to '_JSCreate'.
Reason: All javascript will be coded here. Not only for the radio button. 'Create' is used as reference to the Create.cshtml file in the corresponding Model of the View folder.